source/includes/steps-general-connection-tab.yaml

```diff
@@ -0,0 +1,66 @@
+---
+title: Click on :guilabel:`Advanced Connection Options`.
+level: 4
+ref: compass-advanced-connection
+content: |
+  .. figure:: /images/compass/advanced-connection-options.png
+     :figwidth: 690px
+     :alt: New Advanced Connection Options
+---
+title: Select :guilabel:`General`.
```

Comment:
Maybe "Click the General Tab" ? This is a nit, but "click" feels more appropriate to me for a tab than "select".

Reply:
Done.